File: SplashScreen.java
public class SplashScreen {
    public void display() {
        // Create the splash screen layout
        System.out.println("Welcome to Loan Calculator");
        System.out.println("Press Enter to continue...");

        // Wait for the user to press Enter
        try {
            System.in.read();
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: splash_screen.dart
import 'package:flutter/material.dart';
import 'calculator_screen.dart';

class SplashScreen extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('Loan Calculator App'),
      ),
      body: Center(
        child: Column(
          mainAxisAlignment: MainAxisAlignment.center,
          children: <Widget>[
            Text(
              'Welcome to Loan Calculator',
              style: TextStyle(fontSize: 20),
            ),
            ElevatedButton(
              onPressed: () {
                Navigator.push(
                  context,
                  MaterialPageRoute(builder: (context) => CalculatorScreen()),
                );
              },
              child: Text('Enter'),
            ),
          ],
        ),
      ),
    );
  }
}
